### DISCUSSION TITLE
Search in v2

### DISCUSSION THREAD

**Discussion by FishEconomist** [automadcms/discussions#11] [Q&A]
Hi Automad lovers,

I've installed Automad v2 as an experiment ([Travel by Corsidecape](http://travel.corsidecape.com/)), and I'm wondering how to add a search module: I see it's possible, like on this install of Automad: [Sopcode](http://www.sopcode.com/), but I haven't found a way to do it - and unless I've missed it, it's not in the current documentation.

Any idea?
Thanks
Seb

**Comment by marcantondahmen** [comment]
Hi there, search is depending on theme. In case of the standard theme, setting a proper URL for the `URL Search Results` field in the global settings will enable the search.

**Reply by FishEconomist** [reply]
Okay—I've been able to implement that. Subsequent question: When I type something in the research, I get a 404 result, no matter what. Our friend at Sopcode is in the same situation. Correct me if I'm wrong, but the "Standard v1" and the "Bulma" themes don't have search capabilities. Or did I miss something?

**Reply by marcantondahmen** [reply]
Hi, all official themes have a search function. The process is similar:

1. Define a URL where the results should show up
2. On that result page, choose a template with a pagelist
